From 72000d278ea96b671c0e7a1e6030cedcce38efee Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Maxime=20M=C3=A9r=C3=A9?= <maxime.mere@foss.st.com>
Date: Fri, 30 Aug 2024 14:50:09 +0200
Subject: [PATCH 678/841] feat(fdts): change PKA compatible name
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Change PKA compatible name to match other ST device tree.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 258149bc0f1791f9c51b66c71faa4152acea08ed feat(fdts): change PKA compatible name]
Signed-off-by: Maxime Méré <maxime.mere@foss.st.com>
Change-Id: I5e6f72e68ebc90a114d2310e69d1adfb68da287e
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/401192
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 fdts/stm32mp13xc.dtsi | 2 +-
 fdts/stm32mp13xf.dtsi | 2 +-
 fdts/stm32mp23xc.dtsi | 2 +-
 fdts/stm32mp23xf.dtsi | 2 +-
 fdts/stm32mp25xc.dtsi | 2 +-
 fdts/stm32mp25xf.dtsi | 2 +-
 6 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/fdts/stm32mp13xc.dtsi b/fdts/stm32mp13xc.dtsi
index 4b30c5c32..7fd90e9b0 100644
--- a/fdts/stm32mp13xc.dtsi
+++ b/fdts/stm32mp13xc.dtsi
@@ -17,7 +17,7 @@
 		};
 
 		pka: pka@54006000 {
-			compatible = "st,stm32-pka64";
+			compatible = "st,stm32mp13-pka64";
 			reg = <0x54006000 0x2000>;
 			clocks = <&rcc PKA>;
 			resets = <&rcc PKA_R>;
diff --git a/fdts/stm32mp13xf.dtsi b/fdts/stm32mp13xf.dtsi
index 887c4e006..2a22d23af 100644
--- a/fdts/stm32mp13xf.dtsi
+++ b/fdts/stm32mp13xf.dtsi
@@ -16,7 +16,7 @@
 		};
 
 		pka: pka@54006000 {
-			compatible = "st,stm32-pka64";
+			compatible = "st,stm32mp13-pka64";
 			reg = <0x54006000 0x2000>;
 			clocks = <&rcc PKA>;
 			resets = <&rcc PKA_R>;
diff --git a/fdts/stm32mp23xc.dtsi b/fdts/stm32mp23xc.dtsi
index 7ca6564bd..afeb66d5d 100644
--- a/fdts/stm32mp23xc.dtsi
+++ b/fdts/stm32mp23xc.dtsi
@@ -31,7 +31,7 @@
 		};
 
 		pka: pka@42060000 {
-			compatible = "st,stm32-pka64";
+			compatible = "st,stm32mp13-pka64";
 			reg = <0x42060000 0x2000>;
 			clocks = <&rcc CK_BUS_PKA>;
 			resets = <&rcc PKA_R>;
diff --git a/fdts/stm32mp23xf.dtsi b/fdts/stm32mp23xf.dtsi
index 7ca6564bd..afeb66d5d 100644
--- a/fdts/stm32mp23xf.dtsi
+++ b/fdts/stm32mp23xf.dtsi
@@ -31,7 +31,7 @@
 		};
 
 		pka: pka@42060000 {
-			compatible = "st,stm32-pka64";
+			compatible = "st,stm32mp13-pka64";
 			reg = <0x42060000 0x2000>;
 			clocks = <&rcc CK_BUS_PKA>;
 			resets = <&rcc PKA_R>;
diff --git a/fdts/stm32mp25xc.dtsi b/fdts/stm32mp25xc.dtsi
index dd0036ca3..c58fa9fec 100644
--- a/fdts/stm32mp25xc.dtsi
+++ b/fdts/stm32mp25xc.dtsi
@@ -31,7 +31,7 @@
 		};
 
 		pka: pka@42060000 {
-			compatible = "st,stm32-pka64";
+			compatible = "st,stm32mp13-pka64";
 			reg = <0x42060000 0x2000>;
 			clocks = <&rcc CK_BUS_PKA>;
 			resets = <&rcc PKA_R>;
diff --git a/fdts/stm32mp25xf.dtsi b/fdts/stm32mp25xf.dtsi
index dd0036ca3..c58fa9fec 100644
--- a/fdts/stm32mp25xf.dtsi
+++ b/fdts/stm32mp25xf.dtsi
@@ -31,7 +31,7 @@
 		};
 
 		pka: pka@42060000 {
-			compatible = "st,stm32-pka64";
+			compatible = "st,stm32mp13-pka64";
 			reg = <0x42060000 0x2000>;
 			clocks = <&rcc CK_BUS_PKA>;
 			resets = <&rcc PKA_R>;
-- 
2.47.2

